libs/webb-ui-components/src/components/ThemeToggle/ThemeToggle.tsx

```diff
@@ -0,0 +1,46 @@
+import { MoonLine, SunLine } from '@webb-tools/icons';
+import { useDarkMode } from '../../hooks/useDarkMode';
+
+/**
+ * ThemeToggle (Dark/Light) Component
+ *
+ * ```jsx
+ *  // Example
+ *  <ThemeToggle />
+ * ```
+ */
+export const ThemeToggle = () => {
+  const [isDarkMode, toggleThemeMode] = useDarkMode();
+
+  return (
+    <div
+      className={`relative inline-block w-14 h-8 align-middle select-none transition duration-200 ease-in rounded-full ${
+        isDarkMode ? 'bg-mono-200' : 'bg-blue-10'
+      }`}
+    >
+      <input
+        type="checkbox"
+        name="toggle"
+        id="toggle"
+        className="toggle-checkbox hidden"
+        checked={isDarkMode}
+        onChange={() => toggleThemeMode()}
```

Comment:
The toggle function should be placed at the top-level component, (the `<div></div>` in this case), to avoid having non-clickable components.

![CleanShot 2023-07-05 at 23 14 41](https://github.com/webb-tools/webb-dapp/assets/60747384/a22c49fe-115f-4096-9995-dde5cb4f07a3)